.travis.yml

```diff
@@ -21,7 +21,7 @@ before_install:
 
 script:
  - docker build -f utils/docker/Dockerfile.$DOCKER_IMAGE -t daos/$DOCKER_IMAGE --build-arg NOBUILD=1 --build-arg UID=$UID .
- - docker run -v $PWD:/home/daos/daos:z daos/$DOCKER_IMAGE /bin/bash -c "scons --build-deps=yes install"
+ - docker run -v $PWD:/home/daos/daos:z daos/$DOCKER_IMAGE /bin/bash -c "scons --build-deps=yes install BUILD_TYPE=dev"
```

Comment:
Shouldn't this reflect dev or release, depending on whether the build is supposed to be a release build or not and be secure-by-default?